docs/cluster-deployment/deploy-to-cluster-aws/signup-aws.rst

```diff
@@ -49,17 +49,36 @@ configuration on the CrateDB Cloud end.
 Click on the *Set Up Your Account* button to proceed.
 
 
+.. _signup-aws-contract:
+
+Using the CrateDB Cloud Contract on AWS Marketplace
+===================================================
+
+Alternatively, you can avail yourself of the possibility of an annual contract
+for our cloud service through the CrateDB Cloud Contract. On AWS Marketplace,
+the contract is a separate product for subscription purposes. You can access it
+by going to the `CrateDB Cloud Contract landing page`_.
+
```

Comment:
Can we add a note, that if customer's plan to purchase higher volumes they should get in contact with sales@crate.io first to negotiate a private offer?

Reply:
That is also mentioned on the Subscription Plans page, but I can certainly add it here too